.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "python.analysis.stubPath": ".vscode",
+    "databricks.python.envFile": "${workspaceFolder}/.env",
+    "jupyter.interactiveWindow.cellMarker.codeRegex": "^# COMMAND ----------|^# Databricks notebook source|^(#\\s*%%|#\\s*\\<codecell\\>|#\\s*In\\[\\d*?\\]|#\\s*In\\[ \\])",
+    "jupyter.interactiveWindow.cellMarker.default": "# COMMAND ----------",
+    "python.testing.pytestArgs": [
+        "."
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "files.exclude": {
+        "**/*.egg-info": true,
+        "**/__pycache__": true,
+        ".pytest_cache": true,
+    },
+    "debugpy.debugJustMyCode": false,
+    "editor.formatOnSave": true,
+    "[python]": {
+        "editor.defaultFormatter": "ms-python.black-formatter",
+        "editor.formatOnSave": true,
+    }
+}
```

Comment:
Please remove, no need to include this in the examples repo. Can be added to the repo wide ignore file.

Reply:
Why? This file helps make sure that the templates and other examples comply to Python coding conventions. It prevented some issues before my venv got broken.

Reply:
What kind of issues? Most of these settings don't make sense in this context.

Reply:
I moved it to a separate PR: https://github.com/databricks/bundle-examples/pull/49#issue-2710981854